datasets/cl/info_probidad/cl_info_probidad.yml

```diff
@@ -0,0 +1,1047 @@
+        is_pep: false
+      - match: "DIRECTORA SERVICIOS INCORPORADOS"
+        is_pep: false
+      - match: "DOCENTE"
+        is_pep: false
+      - match: "DOCENTE AULA"
+        is_pep: false
+      - match: "DOCENTE DIRECTIVO"
+        is_pep: false
+      - match: "EDUCADOR (A) DIFERENCIAL"
+        is_pep: false
+      - match: "EDUCADORA DE PARVULO"
+        is_pep: false
+      - match: "EMBAJADOR"
+        is_pep: true
+      - match: "EMBAJADOR PRIMERA CATEGORÍA EXTERIOR"
+        is_pep: true
+      - match: "EMBAJADORA"
+        is_pep: true
+      - match: "ENCARGADA DE ADQUISICIONES"
+        is_pep: false
+      - match: "ENCARGADA DE BODEGA  MUNICPAL"
+        is_pep: false
+      - match: "ENCARGADA DE INVENTARIO"
+        is_pep: false
+      - match: "ENCARGADA UNIDAD DE ADQUISICIONES"
+        is_pep: false
+      - match: "ENCARGADO"
+        is_pep: false
+      - match: "ENCARGADO DE INFRAESTRUCTURA"
+        is_pep: false
+      - match: "ENCARGADO DE PLANIFICACION"
+        is_pep: false
+      - match: "ENCARGADO DE UNIDAD"
+        is_pep: false
+      - match: "ENCARGADO DEPARTAMENTO DE EJECUCION"
+        is_pep: false
+      - match: "ENCARGADO INFRAESTRUCTURA ESCUELAS"
+        is_pep: false
+      - match: "ENCARGADO OFICINA DESARROLLO ECONOMICO LOCAL"
+        is_pep: false
+      - match: "ENCARGADO VIVIENDA"
+        is_pep: false
+      - match: "ENCARGADO ZONAL"
+        is_pep: false
+      - match: "ENFERMERA"
+        is_pep: false
+      - match: "ENFERMERA (O)"
+        is_pep: false
+      - match: "ENFERMERO"
+        is_pep: false
+      - match: "EX ALCALDE"
+        is_pep: false
+      - match: "EX ALCALDESA"
+        is_pep: false
+      - match: "EX CONCEJAL"
+        is_pep: false
+      - match: "EX-CONCEJAL"
+        is_pep: false
+      - match: "EXPERTO"
+        is_pep: false
+      - match: "FISCAL "
+        is_pep: false
+      - match: "FISCAL ADJUNTO"
+        is_pep: false
+      - match: "FISCAL ADJUNTO JEFE"
+        is_pep: false
+      - match: "FISCAL JUDICIAL"
+        is_pep: false
+      - match: "FISCAL NACIONAL"
+        is_pep: false
+      - match: "FISCAL NACIONAL ECONÓMICO"
+        is_pep: false
+      - match: "FISCAL REGIONAL"
+        is_pep: false
+      - match: "FISCALIZADOR"
+        is_pep: false
+      - match: "FISCALIZADOR/A"
+        is_pep: false
+      - match: "FISCALIZADORES"
+        is_pep: false
+      - match: "FONOAUDIOLOGO"
+        is_pep: false
+      - match: "FUNCIÓN FISCALIZACIÓN O APOYO A FISCALIZACIÓN "
+        is_pep: false
+      - match: "GENERAL DE EJERCITO"
+        is_pep: false
+      - match: "GERENTE"
+        is_pep: false
+      - match: "GERENTE GENERAL"
+        is_pep: false
+      - match: "GERENTE GENERAL INTERINO"
+        is_pep: false
+      - match: "GOBERNADOR "
+        is_pep: false
+      - match: "GOBERNADOR REGIONAL"
+        is_pep: false
+      - match: "GOBERNADORA REGIONAL"
+        is_pep: false
```

Comment:
If these are governors, as in the heads of regional government, I would say they're PEPs

Reply:
Indeed they are (done in https://github.com/opensanctions/opensanctions/pull/512/commits/ebebc840651620c6b3262f48e6e137854f2db6d2).

From what I could gather they are distinct because only after 2021 the (regional) governors started to be elected directly by the citizen vote instead of being appointed by the president.